homeassistant/helpers/entity_platform.py

```diff
@@ -315,6 +315,13 @@ async def async_add_entities(
                 self.platform_name,
                 timeout,
             )
+        except Exception:
```

Comment:
Do we need to catch `asyncio.CancelledError` first and just reraise that without log message? Until Python 3.8 `asyncio.CancelledError` inherits `Exception` instead of `BaseException`.

Reply:
Ok, let's skip it then.

Reply:
Let's not add it, as 3.7 is dropped in 2 months.